src/plugins/blocksmith/components/blocksmith/index.jsx

```diff
@@ -1375,7 +1378,7 @@ class Blocksmith extends React.Component {
    * behavior inside the sidebar - if this is a click on the sidebar button itself then e
    * will be defined.
    *
-   * @param {SyntheticKeyboardEvent} e - a synthetic keyboard event
+   * @param {event} e
```

Comment:
@cohencodes this is the Synthetic one, it is called by a react onClick in the sidebar